### DISCUSSION TITLE
Metamask account do not

### DISCUSSION THREAD

**Discussion by Belf33** [Synthetixio/synpress#525] [General]
if I run via "synpress run --configFile=synpress.config.js" metamask set up each time from scratch but
if I run "synpress open --configFile=synpress.config.js" I see that in tests Metamask have already installed.
as I understand  cy.resetMetamaskAccount() do not work in Open way just in run?
if I add  after block with resetMetamaskAccount() it is still doesn't work.

<img width="954" alt="Screenshot 2022-10-03 at 17 25 56" src="https://user-images.githubusercontent.com/20465760/193602151-860ff571-0c6a-49b8-9687-bdcade547e8c.png">
<img width="952" alt="Screenshot 2022-10-03 at 17 26 14" src="https://user-images.githubusercontent.com/20465760/193602172-871b5721-3949-4456-b1de-ef722907ed94.png">

**Comment by drptbl** [comment]
Hey,

`synpress open` is not (fully) supported yet, please use `synpress run` instead.

Thanks,
Jakub.
